smoketest/tests/send_token.rs

```diff
@@ -89,7 +89,7 @@ async fn send_token() {
 
 	assert_eq!(receipt.status.unwrap().as_u64(), 1u64);
 
-	let wait_for_blocks = 100;
+	let wait_for_blocks = 1000;
```

Comment:
This increase will cause test failures to be super slow. Is it necessary?

Reply:
It's nessessary for testnet/production as the finality latency is about 1 hour there, so we need to wait a bit longer.

Maybe better to load from some env and decrease it for local setup.

Reply:
> Maybe better to load from some env and decrease it for local setup.

Good idea!

Reply:
https://github.com/Snowfork/snowbridge/pull/1291/commits/2d97157e224d69db5a35de1fad87b211c0ab3560